tools/torchci/td/get_merge_base_info.py

```diff
@@ -87,24 +82,29 @@ def upload_merge_base_info(shas: List[str]) -> None:
                 f"git show --no-patch --format=%ct {merge_base}"
             )
             timestamp = datetime.utcfromtimestamp(int(unix_timestamp)).isoformat() + "Z"
-            docs.append(
-                {
-                    "sha": sha,
-                    "merge_base": merge_base,
-                    "changed_files": changed_files.splitlines(),
-                    "merge_base_commit_date": timestamp,
-                    "repo": "pytorch/pytorch",
-                }
+            data = {
+                "sha": sha,
+                "merge_base": merge_base,
+                "changed_files": changed_files.splitlines(),
+                "merge_base_commit_date": timestamp,
+                "repo": "pytorch/pytorch",
+                "_id": f"pytorch-pytorch-{sha}",
+            }
+            body = io.StringIO()
+            json.dump(data, body)
+            S3_RESOURCE.Object(
+                f"ossci-raw-job-status",
+                f"merge_bases/pytorch/pytorch/{sha}",
```

Comment:
Probably not needed, but I have seen ClickHouse S3 ClickPipe confuses the input without the `.gzip` filename extension. Adding the correct extension and it works fine.

Reply:
Changed